### DISCUSSION TITLE
How to just call <Button bg='primary'></Button> when given such a colorscheme

### DISCUSSION THREAD

**Discussion by bonface221** [chakra-ui/chakra-ui#8522] [Q&A] [answered]
export const theme = extendTheme({
  fonts: {
    heading: epilogue.style.fontFamily,
  },
  colors: {
    brand: {
      black: "#000000",
      white: "#FFFFFF",
    },
  
    primary: {
      100: "#CEF7FC",
      200: "#9EEAF9",
      300: "#6CD0EE",
      400: "#46B2DE",
      500: "#1288C9",
      600: "#0D6AAC",
      700: "#094F90",
      800: "#053874",
      900: "#032860",
      default: "#1288C9",
    },
    
    secondary: {
      100: "#FEF8D5",
      200: "#FDEFAB",
      300: "#F9E181",
      400: "#F4D361",
      500: "#EEBE2F",
      600: "#CC9D22",
      700: "#AB7E17",
      800: "#8A610E",
      900: "#724C09",
      default: "#EEBE2F",
    },
  },
});

**Comment by segunadebayo** [comment, marked as answer]
Use `colorPalette="primary"` after you define a primary palette.
